package.json

```diff
@@ -84,8 +84,8 @@
     "jwt-decode": "^3.1.2",
     "layer-manager": "^3.0.11",
     "luma.gl": "^7.3.2",
-    "mapbox-gl": "^1.13.0",
     "mapbox-gl-compare": "^0.4.0",
+    "maplibre-gl": "^1.15.3",
```

Comment:
This is a version from Apr 14, 2022 (https://github.com/maplibre/maplibre-gl-js/releases/tag/v1.15.3). Maybe once we are able to migrate to newer libraries and refactor our codebase?

Reply:
After experiencing unusual behavior with the latest version of Maplibre-GL, I decided to revert to version 1.15.3. During the migration of our page to app Router, we'll do the refactoring of map component's